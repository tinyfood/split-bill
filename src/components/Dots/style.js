import styled from 'styled-components'
import media from '../../styles/templates/'

export const DotsStyled = styled.div`
  display: flex;
  justify-content: space-between;
  max-width: 44px;
  margin-bottom: 14px;

  ${media.phone`
    display: none;
  `}
`

export const Dot = styled.div`
  width: 10px;
  height: 10px;
  border-radius: 50%;
  background-color: ${props =>
    props.red
      ? '#ff5c5c'
      : props.orange
      ? '#ffbd4c'
      : props.green
      ? '#19cc32'
      : 'none'};
`
