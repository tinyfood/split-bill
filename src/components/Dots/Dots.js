import React from 'react'
import { DotsStyled, Dot } from './style'

const Dots = () => (
  <DotsStyled>
    <Dot red />
    <Dot orange />
    <Dot green />
  </DotsStyled>
)

export default Dots
