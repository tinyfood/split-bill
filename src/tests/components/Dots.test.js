import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import Dots from '../../components/Dots/Dots'

test('should render Dots correctly', () => {
  const wrapper = shallow(<Dots />)
  expect(toJSON(wrapper)).toMatchSnapshot()
})
