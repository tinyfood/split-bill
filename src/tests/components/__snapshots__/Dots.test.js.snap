// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Dots correctly 1`] = `
<styled.div>
  <styled.div
    red={true}
  />
  <styled.div
    orange={true}
  />
  <styled.div
    green={true}
  />
</styled.div>
`;
